Google.Ads.GoogleAds/examples/HotelAds/AddPerformanceMaxForTravelGoalsCampaign.cs

```diff
@@ -0,0 +1,1037 @@
+                "Google Places database. See " +
+                "https://developers.google.com/places/web-service-place-id to learn more.")]
+            public string PlaceId { get; set; }
+        }
+
+         /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            Options options = ExampleUtilities.ParseCommandLine<Options>(args);
+
+            AddPerformanceMaxForTravelGoalsCampaign codeExample =
+                new AddPerformanceMaxForTravelGoalsCampaign();
+            Console.WriteLine(codeExample.Description);
+            codeExample.Run(new GoogleAdsClient(), options.CustomerId, options.PlaceId);
+        }
+
+        // Minimum requirements of assets required in a Performance Max asset group.
+        // See https://developers.google.com/google-ads/api/docs/performance-max/assets for details.
+        private Dictionary<AssetFieldType, int> MIN_REQUIRED_TEXT_ASSET_COUNTS =
+            new Dictionary<AssetFieldType, int>()
+            {
+                { AssetFieldType.Headline, 3},
+                { AssetFieldType.LongHeadline, 1},
+                { AssetFieldType.Description, 2},
+                { AssetFieldType.BusinessName, 1},
+            };
+
+        private Dictionary<AssetFieldType, int> MIN_REQUIRED_IMAGE_ASSET_COUNTS =
+            new Dictionary<AssetFieldType, int>()
+            {
+                { AssetFieldType.MarketingImage, 1},
+                { AssetFieldType.SquareMarketingImage, 1},
+                { AssetFieldType.Logo, 1},
+            };
+
+
+        // Texts and URLs used to create text and image assets when the TravelAssetSuggestionService
+        // doesn't return enough assets required for creating an asset group.
+        private Dictionary<AssetFieldType, List<string>> DEFAULT_TEXT_ASSETS_INFO =
+            new Dictionary<AssetFieldType, List<string>>()
+            {
+                { AssetFieldType.Headline, new List<string>()
+                    {
+                        "Hotel", "Travel Reviews", "Book travel"
+                    }
+                },
+                { AssetFieldType.LongHeadline, new List<string>() { "Travel the World" } },
+                { AssetFieldType.Description, new List<string>()
+                    {
+                        "Great deal for your beloved hotel",
+                        "Best rate guaranteed"
+                    }
+                },
+                { AssetFieldType.BusinessName, new List<string>() { "Interplanetary cruises" } },
+            };
+
+        private Dictionary<AssetFieldType, List<string>> DEFAULT_IMAGE_ASSETS_INFO =
+            new Dictionary<AssetFieldType, List<string>>()
+            {
+                { AssetFieldType.MarketingImage, new List<string>()
```

Comment:
nit: I think 

{ AssetFieldType.MarketingImage, new [] { "https://gaagl.page.link/Eit5" } },

might work as an alternative.

Reply:
Nope, doesn't work, left it as it was.